src/SearchWrapper.cpp

```diff
@@ -366,6 +366,19 @@ void
     }
   }
 
+// [[Rcpp::export]]
+Rcpp::LogicalVector
+  RangeProcSupport()
```

Comment:
Does R have a convention for these kinds of functions? A lot of other languages you'd call it `hasRangeProcSupport` or similar.